lldb/test/API/tools/lldb-dap/stepInTargets/TestDAP_stepInTargets.py

```diff
@@ -0,0 +1,66 @@
+"""
+Test lldb-dap stepInTargets request
+"""
+
+import dap_server
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+import lldbdap_testcase
+from lldbsuite.test import lldbutil
+
+
+class TestDAP_stepInTargets(lldbdap_testcase.DAPTestCaseBase):
+    @skipIf(archs=no_match(["x86_64"]))  # ARM flow kind is not supported yet.
```

Comment:
mention that this doesn't work on arm because of lack of InstructionControlFlowKind support